### ISSUE TITLE
It seems there is a problem getting Mycroft to stop speaking

### ISSUE THREAD

**Issue by andlo** [andlo/fairytalez-skill#10]
It seems that there is a problem getting Mycroft to stop speaking.

It looks like it is mycroft related ass the skill has implemented posibility to stop. But the voice queue isnt stopping as expected.

**Comment by andlo** [comment]
It seems it is a problem in the mycroft messagebus. Waiting on results from Åke-mycroft

**Comment by andlo** [comment]
Just tested Åke's change to the messagebus, and it seems to fix the stop issue.
Now waiting for it to get merged into mycroft-core

**Comment by vwbusguy** [comment]
Was this merged in?  I couldn't get mycroft to stop when I tried today.  When I asked mycroft to 'stop reading" or "be quiet", it just picked a different fairy tale or kept reading.
